docs/getting_started/index.rst

```diff
@@ -1,11 +1,46 @@
 Getting started
 ===============
 
+Wagtail is built on the `Django web framework <https://www.djangoproject.com/>`_, so this document assumes you've already got the essentials installed. But if not, those essentials are:
+
+ * `Python <https://wiki.python.org/moin/BeginnersGuide/Download>`_
+ * `pip <https://pip.pypa.io/en/latest/installing.html>`_ (Note that pip is included by default with Python 2.7.9 and later and Python 3.4 and later)
+
+We'd also recommend:
```

Comment:
By the way, I'm not opposed to removing the references to virtualenv. But at the same time, I don't want us to assume that everyone who reads this doc is a Python/Django wizard already.